docs/interactive-mode.rst

```diff
@@ -59,3 +59,41 @@ the corresponding number of bytes are read over stdin after the json object and
     {"_type": "completed", "command": "generate-tags"}
 
 .. _json lines: http://jsonlines.org/
+
+.. _sandbox-submode:
+
+sandbox submode
+--------------------------
+..
+	NOT REVIEWED YET
+
+``sandbox`` submode can be used with ``--_interactive=sandbox``.  This
+submode will activate a sandbox, to this limits the damage that the
+can be achieved when exploiting a buffer overflow in Universal-ctags.
+
+In the sandbox submode ctags can generate tags only for non-fileio
+requests because ctags have to use open system call to handle fileio
+requests. The open system call is limited in the sandbox.
+
+This feature uses seccomp-bpf, and is only supported on Linux.
+Whether the sandbox submode is available or not depends on build-time
+and run-time environments. libseccomp is needed in build-time. If a
+ctags executable linked in the build-time, ``sandbox`` is listed in the
+output of ``--list-features`` option. About the run-time, it is up to
```

Comment:
seccomp-bpf was introduced in 2012 which is a while ago. I think you can just drop the remark about runtime.